front/components/assistant_builder/submitAssistantBuilderForm.ts

```diff
@@ -70,13 +89,7 @@ export async function submitAssistantBuilderForm({
             name: a.name,
             description: a.description,
             query: a.type === "RETRIEVAL_SEARCH" ? "auto" : "none",
-            relativeTimeFrame:
-              a.type === "RETRIEVAL_EXHAUSTIVE"
-                ? {
-                    duration: a.configuration.timeFrame.value,
-                    unit: a.configuration.timeFrame.unit,
-                  }
-                : "auto",
```

Comment:
what is "auto" that has been removed ?

Reply:
hasn't been removed (see above line 65). It's just what we use for regular retrieval

Reply:
(i.e the model decides the time frame)